setup.py

```diff
@@ -1,9 +1,9 @@
-import setuptools
+from setuptools import setup, find_packages
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-setuptools.setup(
+setup(
     name="speakleash",
     version="0.2.0",
```

Comment:
Shouldn't the version number be updated?

Reply:
Version number will be updated in the day of the deployment.